geniza/common/migrations/0005_create_staff_user.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.6 on 2021-04-05 13:20
+
+from django.conf import settings
+from django.db import migrations
+
+
+def create_staff_user(apps, schema_editor):
```

Comment:
staff seems misleading/inaccurate for this, why not just call it geniza lab user?